### PR TITLE
[Storage] Make SignedResource not required in listSericeSAS

### PR DESCRIPTION
**Pull request by blueww**
## Description

SignedResource not required in listSericeSAS from server, so set is as not required in swagger/SDK.
Also add description to operation to fix swagger validate failure. (this failure not happen before, might cause by autorest upgrade)

Swagger pass review and merged in https://github.com/Azure/azure-rest-api-specs/pull/3463 
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/_metadata/storage_resource-manager.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
 1) azure-rest-api-specs repository information
 GitHub user: azure
 Branch:      master
-Commit:      37ca9e71d6c89b46ab990eb064ddb70308fd5824
+Commit:      1e2f0dae5c44d55b642d27232f9dcd7c0876488a
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\weiwei\AppData\Roaming\npm `-- autorest@2.0.4280
```
Comment (dsgouda) [inline src/SDKs/_metadata/storage_resource-manager.txt]: Please pull down the latest changes from psSdkJson6, run `msbuild build.proj` and regenerate the code

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please bump the version number to `8.1.1`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Since this is a non breaking change, please bump the version to `8.1.1-preview`

**Comment by blueww** [comment]
@dsgouda 

I have update he PR by rebase from psSdkJson6, run msbuild build.proj and regenerate the code.
O also upgrade to 8.1.1, but we won't release SRP SDK for this. will release  with next new feature.

**Comment by dsgouda** [comment]
@blueww in case you are not planning to release this version please revert the version back to `8.1.0-preview`

**Comment by blueww** [comment]
@dsgouda 

I have reverted the version to 8.1.0

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**